BitwardenShared/UI/Auth/UpdateMasterPassword/UpdateMasterPasswordProcessorTests.swift

```diff
@@ -265,6 +267,32 @@ class UpdateMasterPasswordProcessorTests: BitwardenTestCase {
         XCTAssertEqual(subject.state.masterPassword, "password")
     }
 
+    /// `receive(_:)` with `.masterPasswordChanged(_:)` updates the password strength score based on
+    /// the entered password.
+    @MainActor
+    func test_receive_masterPasswordChanged_updatesPasswordStrength() {
```

Comment:
🎨 Could you add another test for when `authRepository.passwordStrength(...)` throws?